### PR TITLE
Send big endian values to be compatible with StackState 4.4 (STAC-14978)

### PR DESCRIPTION
**Pull request by Q69K**

### CODE REVIEW

**Review by julianosk — APPROVED** [review]
LGTM

File: model/message_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+		Type:           TypeCollectorProc,
+		SubscriptionID: 0,
+		OrgID:          0,
+		Timestamp:      1638527655412,
+	}
+	headerBytes, err := encodeHeader(header)
+	assert.NoError(t, err)
+	headerB64 := base64.StdEncoding.EncodeToString(headerBytes)
+
+	assert.EqualValues(t, "AwIMAAAAAAAAAAF9f9vd9A==", headerB64)
```
Comment (julianosk) [inline model/message_test.go:38]: These bytes encoded to string are the same you used in the stackstate PR, right? Should we add a comment so we keep them in sync?

**Review by julianosk — APPROVED** [review]
LGTM

File: model/message_test.go (hunk 1 above)
Comment (Q69K) [reply model/message_test.go:42]: 👍

**Pull request merged**